packages/stage-ui/src/components/FirstTimeSetup/FirstTimeSetupDialog.vue

```diff
@@ -0,0 +1,365 @@
+          <h3 class="mb-4 text-lg text-neutral-800 font-medium dark:text-neutral-100">
+            {{ t('configureProvider', { provider: selectedProvider.localizedName }) }}
+          </h3>
+
+          <div class="space-y-4">
+            <!-- API Key Input -->
+            <div v-if="needsApiKey">
+              <label class="mb-2 block text-sm text-neutral-700 font-medium dark:text-neutral-300">
+                API Key
+                <span class="text-red-500">*</span>
+              </label>
+              <Input
+                v-model="apiKey"
+                type="password"
+                :placeholder="getApiKeyPlaceholder(selectedProvider.id)"
+                class="w-full"
+                :required="true"
+              />
+            </div>
+
+            <!-- Base URL Input -->
+            <div v-if="needsBaseUrl">
+              <label class="mb-2 block text-sm text-neutral-700 font-medium dark:text-neutral-300">
+                Base URL
+              </label>
+              <Input
+                v-model="baseUrl"
+                type="url"
+                :placeholder="getBaseUrlPlaceholder(selectedProvider.id)"
+                class="w-full"
+              />
+            </div>
+
+            <!-- Account ID for Cloudflare -->
+            <div v-if="selectedProvider.id === 'cloudflare-workers-ai'">
+              <label class="mb-2 block text-sm text-neutral-700 font-medium dark:text-neutral-300">
+                Account ID
+                <span class="text-red-500">*</span>
+              </label>
+              <Input
+                v-model="accountId"
+                type="text"
+                placeholder="Account ID"
+                class="w-full"
+                :required="true"
+              />
+            </div>
+          </div>
+
+          <!-- Validation Status -->
+          <div v-if="validationMessage" class="mt-4">
+            <div
+              class="flex items-center rounded-lg p-3" :class="[
+                isValidating
+                  ? 'bg-blue-50 text-blue-700 dark:bg-blue-900/30 dark:text-blue-300'
+                  : isValid
+                    ? 'bg-green-50 text-green-700 dark:bg-green-900/30 dark:text-green-300'
+                    : 'bg-red-50 text-red-700 dark:bg-red-900/30 dark:text-red-300',
+              ]"
+            >
+              <div
+                class="mr-2 text-lg" :class="[
+                  isValidating
+                    ? 'i-mdi:loading animate-spin'
+                    : isValid
+                      ? 'i-mdi:check-circle'
+                      : 'i-mdi:alert-circle',
+                ]"
+              />
+              {{ validationMessage }}
+            </div>
+          </div>
+        </div>
+
+        <!-- Action Buttons -->
+        <div class="flex flex-col-reverse gap-3 sm:flex-row sm:justify-end">
+          <Button
+            variant="secondary"
+            :label="t('firstTimeSetup.skipForNow')"
```

Comment:
```suggestion
            :label="t('firstTimeSetup.skipForNow')"
```

So many, add `settings` for this.